db/range_tombstone_fragmenter.cc

```diff
@@ -17,22 +17,6 @@
 
 namespace rocksdb {
 
-namespace {
-
-struct ParsedInternalKeyComparator {
```

Comment:
I think this struct is still necessary. The error is coming from the fact that an identically named struct is declared in a non-anonymous namespace in another file. In fact, both structs are identical (and I wrote both versions in separate PRs but forgot to make a single definition for it; sorry about the oversight). So I think the real solution would be to move this struct to another file (maybe `db/dbformat.h`) and delete the identical definition in `db/range_del_aggregator.cc`.

Reply:
yeah just realized my current fix is wrong, will move ParsedInternalKeyComparator definition to dbformat.h